### PR TITLE
Inform where to add/change the value

### PR DESCRIPTION
**Pull request by HamDerAndrew**
As a newcomer it might not be obvious where to change/add this value.

I read through the page and we don't mention `appsettings.json` anywhere, so I added it 😄

### CODE REVIEW

File: Umbraco-Cloud/Deployment/Content-Transfer/index.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ For Deploy to handle Forms data as content, you'll need to add the following set
 
 #### Umbraco version 9 and above
 
-For Deploy to handle Forms data as content, you'll need to ensure the `TransferFormsAsContent` setting is set to `true`:
+For Deploy to handle Forms data as content, you'll need to ensure the `TransferFormsAsContent` setting is set to `true` in the `appsettings.json`file:
```
Comment (sofietoft) [inline Umbraco-Cloud/Deployment/Content-Transfer/index.md:59]: The sentence was being a bit wordy. How about something like this instead? 🤔 

```suggestion
For Deploy to handle Forms data as content, the `TransferFormsAsContent` setting found in the `appsettings.json` file needs to be `true`:
```

**Review by sofietoft — COMMENTED** [review]
Great addtition to the article @HamDerAndrew ! 👏 
Thanks for helping clear this up!

I've added a suggestion to change the structure of the sentence a bit.
Let me know what you think!

File: Umbraco-Cloud/Deployment/Content-Transfer/index.md (hunk 1 above)
Comment (HamDerAndrew) [reply Umbraco-Cloud/Deployment/Content-Transfer/index.md:59]: Looks good @sofietoft 😄

**Comment by sofietoft** [comment]
Great!
Thanks again for the contrib @HamDerAndrew ! 💪 

I'll merge it in right away!
